docfx/doxygen2references.cc

```diff
@@ -48,10 +48,8 @@ std::list<Reference> ExtractReferences(YamlContext const& ctx,
   if (!IncludeInPublicDocuments(ctx.config, node)) return {};
 
   auto const name = std::string_view{node.name()};
-  if (name == "innernamespace") {
-    auto const uid = std::string{node.attribute("refid").as_string()};
-    return {Reference(uid, node.child_value())};
-  }
+  // Skip <innernamespace> elements. They are listed in ToC (the left-side
+  // navigation).
```

Comment:
Consider adding the following to be explicit
```cc
if (name == "innernamespace") return {};
```

Reply:
Well, you know better than I whether we should `RecurseReferences` for this case or not.

Reply:
Done.